frontend/src/model/Wizard.ts

```diff
@@ -8,35 +10,38 @@ export type Wizard = {
 }
 
 export type NewWizard = {
-    name: string
+    name: string,
+    gender: Gender | string,
+    attribute1: Attribute1 | string,
+    attribute2: Attribute2 | string,
+    attribute3: Attribute3 | string,
 }
 
 export enum Gender {
-    WITCHER = "WITCHER",
-    WITCH = "WITCH",
+    WITCHER = "Zauberer",
+    WITCH = "Hexe",
     MUGGLE = "MUGGLE",
     HALFBLOOD = "HALFBLOOD",
     SQUIB = "SQUIB"
 }
 
-
 export enum Attribute1 {
```

Comment:
Variable names should aways be in english.
You can still display the german text but the variable/enum/... name should be english